docs/content/en/docs/Coding Milestones/PoC2023q1/environments/dev-env.md

```diff
@@ -6,6 +6,479 @@ description: >-
  
 ---
 
-## Heading
+##Get Started Using the KCP-Edge Development Environment.
 
-under construction - coming soon
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+#### 1. Create some edge infrastructure (edge pclusters):
+
+  - For example: create a kind cluster
+
+    ```bash
+    kind create cluster --name florin
+    ``` 
+    
+#### 2. Deploy the KCP-Edge platform:
+    
+  * Step-1: Clone this repo:
+
+    ```bash
+    git clone -b dev-env-v2 https://github.com/dumb0002/edge-mc.git
+    ```
+
+  * Step-2: change into the following directory path:
+
+    ```bash
+      cd edge-mc/environments/dev-env
+    ```
+
+  * Step-3: Deploy kcp-edge
+
+    ```bash
+    ./install_edge-mc.sh --stage 0
+    ```
+
+    This will start `kcp` and create/deploy the following components:
+
+    - 4 kcp workspaces: edge service provider workspace (`espw`), inventory management workspace (`imw`) and workload management workspace (`wmw`) under my-org workspace
```

Comment:
(A) Typos here on "imw(-1)" and "wmw(-1)".

(B) The inventory management workspace is not part of the edge platform, it is part of the usage of the edge platform.

(C) The WMW is not part of the edge platform, it is part of the usage of that platform.